### DISCUSSION TITLE
IP Addresses Role

### DISCUSSION THREAD

**Discussion by JHarrison712** [netbox-community/netbox#12265] [Help Wanted!] [answered]
Can anyone help me understand these roles and when to use them?

- Loopback
- Secondary
- Anycast
- VIP
- VRRP
- HSRP
- GLBP

**Comment by candlerb** [comment, marked as answer]
They're just labels, and are documented [here](https://docs.netbox.dev/en/stable/models/ipam/ipaddress/#role). If you are not using any of these features in your network then you can ignore them, as the default is for the role to be unset.
